sdk/language-support/ts/daml-types/index.ts

```diff
@@ -245,19 +245,31 @@ const registeredTemplates: { [key: string]: Template<object> } = {};
  */
 export const registerTemplate = <T extends object>(
   template: Template<T>,
+  packageSpecifiers: string[] = [],
```

Comment:
Allow registering a template with alternative package specifiers. This enables registering with both package id and package name, so that it can be looked up later via either.